modules/core/24-host/channel_keys.go

```diff
@@ -8,6 +8,8 @@ const (
 	KeyChannelUpgradePrefix = "channelUpgrades"
 	KeyChannelRestorePrefix = "restore"
 	KeyUpgradeTimeoutPrefix = "upgradeTimeout"
+
+	KeyUpgradeError = "upgradeError"
```

Comment:
```suggestion
	KeyUpgradeErrorPrefix = "upgradeError"
```

Reply:
why is this called a Prefix if it's not really prefixing anything?